givitsite/coordinate/views.py

```diff
@@ -1,3 +1,131 @@
-from django.shortcuts import render
+import sys
+import os
+from django.shortcuts import render, redirect
+from django.http import HttpRequest, HttpResponse, Http404
+from django.contrib.auth.models import User
+from friendreq.models import ItemsFound
+from friendreq.models import ItemRequest
+from django.conf import settings
+from . models import CoordinatedItems
+
 
 # Create your views here.
+def coordinator_create_view(request):
+    
+    if request.method == 'GET':
+        itemsToCoordinate = []
+        matchedItems = ItemsFound.objects.filter(match=True)
+        openRequests = ItemRequest.objects.filter(status='open')
+        users = User.objects.all()
+        
+
+        # mock data for testing:
+
+        # itemsToCoordinate.append({
+        #     "item":'chair',
+        #     "pickup_location":'Tel Aviv',
+        #     "drop_off_location":'Jerusalem',
+        #     "url": 'https://www.etzmaleh.co.il/Media/Uploads/tn_%D7%9B%D7%99%D7%A1%D7%90_%D7%95%D7%A0%D7%A6%D7%99%D7%94_%D7%AA%D7%9B%D7%9C%D7%AA.jpg_op1_690x920(15).jpg',
+        #     "friend_id":"1234",
+        #     "picture": '',
+        #     "request_id": '1',
+        #     "user_id": '2'
+        # })
+
+        # itemsToCoordinate.append({
+        #     "item":'chair',
+        #     "pickup_location":'Jerusalem',
+        #     "drop_off_location":'Tel Aviv',
+        #     "url": 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRxJI6_VVfFWUPcvozhhY7qNVWaujOLj1zI8A&usqp=CAU',
+        #     "friend_id":"1235",
+        #     "picture": '',
+        #     "request_id": '2',
+        #     "user_id": '3'
+        # })
+
+        # itemsToCoordinate.append({
+        #     "item":'chair',
+        #     "pickup_location":'Jerusalem',
+        #     "drop_off_location":'Tel Aviv',
+        #     "url": 'https://ak1.ostkcdn.com/images/products/13223495/TidySquares-Classic-White-Wood-63-Reachin-Closet-Organizer-Design-2-998c7350-e3b4-4cfd-b3eb-67998079718c.jpg',
+        #     "friend_id":"1235",
+        #     "picture": '',
+        #     "request_id": '2',
+        #     "user_id": '3'
+        # })
+
+        return render (request, 'coordinator.html', {'matchedItems':matchedItems, 'openRequests':openRequests, 'users': users})
+
+    elif request.method == 'POST':
+        save_to_coordinations(request);
+        update_friendreq_db(request);
+        return redirect('/coordinate')
+    
+   
+
+def save_to_coordinations(request):
+
+    print("Times:")
```

Comment:
Use log files instead of print

Reply:
removed the print - it was testing leftovers I did not notice